builtin/settingtypes.txt

```diff
@@ -151,6 +151,16 @@ enable_hotbar_mouse_wheel (Hotbar: Enable mouse wheel for selection) bool true
 #    Requires: keyboard_mouse
 invert_hotbar_mouse_wheel (Hotbar: Invert mouse wheel direction) bool false
 
+#    SDL only: Save key controls as scancodes (position on the keyboard) instead of keycodes (the
+#    symbol marked on the key). This allows using the same physical keys with multiple
+#    keyboard layouts, but the new scancode-based settings can not be parsed by older clients.
```

Comment:
"the new scancode-based settings can not be parsed by older clients."
this does not deserve mentioning since it applies to all new/changed settings

Reply:
Removed in d34dc3520f03af854948ad51d3b609a73c2eb250.